files/image_config/monit/conf.d/sonic-host

```diff
@@ -20,3 +20,7 @@ check process rsyslog with pidfile /var/run/rsyslogd.pid
     start program = "/bin/systemctl start rsyslog.service"
     stop program = "/bin/systemctl stop rsyslog.service"
     if totalmem > 800 MB for 10 times within 20 cycles then restart
+
+check program routeCheck with path "/usr/bin/route_check.py"
```

Comment:
Maybe we should add a comment stating what this script does?

Reply:
Also, can you help understand what would be the log/alert if there is a mismatch? May be it can be put in the description

Reply:
Added comments. Please review.